config/opta-k8s-service-helm/templates/ingress.yaml

```
@@ -36,6 +36,11 @@ metadata:
     nginx.ingress.kubernetes.io/proxy-read-timeout: "3600"
     nginx.ingress.kubernetes.io/proxy-send-timeout: "3600"
     {{- end }}
+    {{- if hasKey $.Values "ingress_extra_annotations" }}
+    {{- range $k, $v := $.Values.ingress_extra_annotations }}
+    {{ $k | quote }}: {{ $v | quote }}
+    {{- end }}
+    {{- end }}
```

```suggestion
    {{- range $k, $v := $.Values.ingressExtraAnnotations }}
    {{ $k | quote }}: {{ $v | quote }}
    {{- end }}
```

- Since there is a default value set, `$.Values` will always have the extra annotations field, so the outer `if` block is unnecessary.
- Updated field to match comment below